src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj

```diff
@@ -5,6 +5,8 @@
     <ProjectReference Include="..\src\Microsoft.XmlSerializer.Generator.csproj" />
   </ItemGroup>
   <PropertyGroup>
+    <ReleaseNotes>https://go.microsoft.com/fwlink/?LinkID=863421</ReleaseNotes>
```

Comment:
Please forward this to https://github.com/dotnet/announcements/issues/39.

Reply:
Done